toolchain/autoupdate_testdata.py

```diff
@@ -11,16 +11,23 @@
 import re
 import subprocess
 import sys
+import os
 from pathlib import Path
 
+directory = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(os.path.join(directory))
```

Comment:
WDYT of using pathlib for this:

```suggestion
sys.path.append(str(Path(__file__).parents[1]))
```

What I'm finding is that `__file__` is always absolute [as of python 3.9](https://docs.python.org/3/whatsnew/3.9.html#other-language-changes), so doesn't need `realpath`

Reply:
Ack.